### PR TITLE
[GSW-SWAP] fix: Fix Swap issue

### PR DESCRIPTION
**Pull request by khiemldk**

### CODE REVIEW

File: packages/web/src/containers/select-token-container/SelectTokenContainer.tsx [hunk 1]
```diff
@@ -37,55 +37,21 @@ const customSort = (a: TokenModel, b: TokenModel) => {
...
-  } else if (a.symbol !== "GNOT" && b.symbol === "GNOT") {
-    return 1;
-  } else if (a.symbol === "GNS" && b.symbol !== "GNS") {
-    return -1;
-  } else if (a.symbol !== "GNS" && b.symbol === "GNS") {
-    return 1;
-  } else {
+
+const handleSort = (list: SortedProps[]) => {
+  const gnot = list.find(a => a.symbol === "GNOT");
```
Comment (jinoosss) [inline packages/web/src/containers/select-token-container/SelectTokenContainer.tsx:42]: How about defining symbolic values as constant values? ex) `GNOT_SYMBOL`

File: packages/web/src/containers/swap-container/SwapContainer.tsx [hunk 2]
```diff
@@ -7,11 +7,28 @@ import { ThemeState } from "@states/index";
 import { useRouter } from "next/router";
 import { useSwapHandler } from "@hooks/swap/use-swap-handler";
 
+import { TokenModel } from "@models/token/token-model";
+const TOKEN_A: TokenModel = {
```
Comment (jinoosss) [inline packages/web/src/containers/swap-container/SwapContainer.tsx:11]: It seems like a good idea to modify the definition of `common/values/token-constant.ts`.

**Review by jinoosss — COMMENTED** [review]
I left a few small comments.

File: packages/web/src/containers/select-token-container/SelectTokenContainer.tsx (hunk 1 above)
Comment (khiemldk) [reply packages/web/src/containers/select-token-container/SelectTokenContainer.tsx]: done

File: packages/web/src/containers/swap-container/SwapContainer.tsx (hunk 2 above)
Comment (khiemldk) [reply packages/web/src/containers/swap-container/SwapContainer.tsx]: done

**Review by dongwon8247 — APPROVED** [review]

**Pull request merged**